core/templates/dev/head/domain/user/UserInfoObjectFactory.ts

```diff
@@ -43,6 +44,7 @@ export class UserInfo {
     this._preferredSiteLanguageCode = preferredSiteLanguageCode;
     this._username = username;
     this._isLoggedIn = isLoggedIn;
+    this._email = email;
```

Comment:
Switch the lines of email and logged in, since in all the other places you have logged in after email.

Reply:
Done